### PR TITLE
[BACKLOG-17366] - I want to support multiple topic ingestion from a s…

### PR DESCRIPTION
**Pull request by Aliaksandr-Kastenka**
…treaming ingestion step(insert variables)

### CODE REVIEW

**Comment by Aliaksandr-Kastenka** [comment]
Goes with https://github.com/pentaho/pentaho-ee/pull/453
@kurtwalker @DFieldFL 
Please review.

File: ui/src/main/java/org/pentaho/di/ui/core/widget/TableView.java [hunk 1]
```diff
@@ -2283,32 +2300,65 @@ private void editCombo( TableItem row, int rownr, int colnr ) {
...
+          StringBuilder buffer = new StringBuilder( table.getItem( rownr ).getText( colnr ) );
+          buffer.insert( position, string );
+          table.getItem( rownr ).setText( colnr, buffer.toString() );
+          int newPosition = position + string.length();
+          edit( rownr, colnr );
+        }
+      };
+
+      combo = new ComboVar( variables, table, SWT.SINGLE | SWT.LEFT | SWT.BORDER, getCaretPositionInterface, insertTextInterface );
+      ( (ComboVar) combo ).addModifyListener( lsModCombo );
```
Comment (kurtwalker) [inline ui/src/main/java/org/pentaho/di/ui/core/widget/TableView.java]: Can we avoid the repeated casting by creating a one local variable with the correct type?

File: ui/src/main/java/org/pentaho/di/ui/core/widget/TableView.java [hunk 2]
```diff
@@ -2283,32 +2300,65 @@ private void editCombo( TableItem row, int rownr, int colnr ) {
...
+    final boolean useVariables = colinfo.isUsingVariables();
+    if ( useVariables ) {
+      GetCaretPositionInterface getCaretPositionInterface = new GetCaretPositionInterface() {
+        @Override
+        public int getCaretPosition() {
+          return 0;
+        }
+      };
+
+      // The text widget will be disposed when we get here
```
Comment (kurtwalker) [inline ui/src/main/java/org/pentaho/di/ui/core/widget/TableView.java]: comment refers to "Text widget"  which I don't think is accurate.

**Comment by kurtwalker** [comment]
I'm going to merge since my suggestions aren't related to correctness.

**Pull request merged**